pkg/apis/crd/v1alpha1/types.go

```diff
@@ -434,6 +434,11 @@ type NetworkPolicyPeer struct {
 	//  Exact FQDNs, i.e. "google.com", "db-svc.default.svc.cluster.local"
 	//  Wildcard expressions, i.e. "*wayfair.com".
 	FQDN string `json:"fqdn,omitempty"`
+	// Select all Pods with the ServiceAccount matched by this field, as
+	// workloads in AppliedTo/To/From fields.
+	// Cannot be set with any other selector.
+	// +optional
+	ServiceAccounts []ServiceAccount `json:"serviceAccounts,omitempty"`
```

Comment:
Why we need a slice for ServiceAccount? I feel we should just support a single ServiceAccount in NetworkPolicyPeer, the same as other fields.

Reply:
Sounds good. Changed.